src/main/kotlin/org/rust/lang/core/macros/MacroExpander.kt

```diff
@@ -205,6 +204,33 @@ class MacroExpander(val project: Project) {
                 )
             }
         }
+
+    /** Returns path from [context] to [def]'s crate */
+    private fun getPathToCrate(context: RsElement, def: RsMacro): String {
+        val defCrate = def.crateRoot ?: return ""
+        if (defCrate == context.crateRoot) return ""
+        val (name, relativeDepth) = getRelativeCrateNameAndDepth(context, defCrate) ?: return ""
+        val prefix = when (relativeDepth) {
```

Comment:
As I understand `extern crate` item with `#[macro_use]` can be only in the crate root (https://doc.rust-lang.org/error-index.html#E0468)
So I think we can drop this complex logic with relative depth and significantly simplify `getPathToCrate` method

Reply:
Fixed